<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClientMainWindow</class>
 <widget class="QMainWindow" name="ClientMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>576</width>
    <height>437</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ClientMainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="labelAdresse">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>10</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Adresse : </string>
    </property>
   </widget>
   <widget class="QLabel" name="labelPort">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>70</y>
      <width>26</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Port</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelEtatConnexion">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>10</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Etat de la connexion :</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelMessageBanque">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>180</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Message de la banque</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonConnexion">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>120</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Connexion</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditAdresse">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>10</y>
      <width>113</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditPort">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>60</y>
      <width>113</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEditEtatConnexion">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>40</y>
      <width>211</width>
      <height>161</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditMessageBanque">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>210</y>
      <width>501</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBoxOperationBanque">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>551</width>
      <height>161</height>
     </rect>
    </property>
    <property name="title">
     <string>Opérations envoyées à la banque</string>
    </property>
    <widget class="QPushButton" name="pushButtonNumeroCompte">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>50</y>
       <width>122</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Numéro de compte</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelAdresseMontant">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>90</y>
       <width>58</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Montant :</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditNumeroCompte">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>50</y>
       <width>113</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditMontant">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>90</y>
       <width>113</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
    <widget class="QWidget" name="">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>20</y>
       <width>102</width>
       <height>80</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QRadioButton" name="radioButtonSolde">
        <property name="text">
         <string>Solde</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButtonDepot">
        <property name="text">
         <string>Dépot</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButtonRetrait">
        <property name="text">
         <string>Retrait</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="pushButtonEnvoi">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>110</y>
       <width>80</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Envoi</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>576</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
